ilastik/experimental/parser/types.py

```diff
@@ -23,30 +34,87 @@ class FeatureMatrix:
     selections: numpy.ndarray
     compute_in_2d: numpy.ndarray
 
+    @staticmethod
```

Comment:
Since it returns an instance of the enclosing class, this could be replaced with `@classmethod`.